AstSemantics.md

```diff
@@ -361,21 +365,18 @@ representing `false`.
   * `int32.sge`: signed greater than or equal
   * `int32.ugt`: unsigned greater than
   * `int32.uge`: unsigned greater than or equal
-  * `int32.clz`: count leading zeroes (defined for all values, including zero)
-  * `int32.ctz`: count trailing zeroes (defined for all values, including zero)
-  * `int32.popcnt`: count number of ones
-
-Explicitly signed and unsigned operations trap whenever the result cannot be
-represented in the result type. This includes division and remainder by zero,
-and signed division overflow (`INT32_MIN / -1`). Signed remainder with a
-non-zero denominator always returns the correct value, even when the
-corresponding division would trap. Signed-less operations never trap.
+  * `int32.clz`: sign-agnostic count leading zeroes (defined for all values, including zero)
+  * `int32.ctz`: sign-agnostic count trailing zeroes (defined for all values, including zero)
+  * `int32.popcnt`: sign-agnostic count number of ones
 
 Shifts interpret their shift count operand as an unsigned value. When the shift
 count is at least the bitwidth of the shift, `shl` and `shr` produce `0`, and
 `sar` produces `0` if the value being shifted is non-negative, and `-1`
 otherwise.
 
+All comparison operations yield 32-bit integer results with `1` representing
+`true` and `0` representing `false`.
```

Comment:
This is pre-existing text that's just getting moved here. Incidentally, I am also contemplating proposing a non-storable bool type :-).